docs/reference/elasticsearch/mapping-reference/semantic-text.md

```diff
@@ -96,6 +102,32 @@ PUT my-index-000003
 }
 ```
 
+### Using ELSER on EIS
+
+```{applies_to}
+stack: preview 9.1
+serverless: preview
+```
+
+If you use the preconfigured `.elser-2-elastic` endpoint that utilizes the ELSER model as a service ([ELSER on EIS](docs-content://explore-analyze/elastic-inference/eis.md#elser-on-eis)), you can
```

Comment:
consider expanding to Elastic Inference Service (EIS) here unless already defined earlier